internal/civisibility/utils/ci_providers.go

```diff
@@ -100,6 +101,12 @@ func normalizeTags(tags map[string]string) {
 	if tag, ok := tags[constants.GitTag]; ok && tag != "" {
 		tags[constants.GitTag] = normalizeRef(tag)
 	}
+	if tag, ok := tags[constants.GitPrBaseBranch]; ok && tag != "" {
+		if strings.Contains(tag, "refs/tags") || strings.Contains(tag, "origin/tags") || strings.Contains(tag, "refs/heads/tags") {
```

Comment:
Why do we need this `if` block?
If it evaluates to false, we still set `tags[constants.GitPrBaseBranch] = normalizeRef(tag)` in line 108 anyway

Reply:
that's a good question I've just copied the one for Branch, I'll check